reference/extensions/custom_commands.rst

```diff
@@ -120,6 +120,46 @@ The command call looks like :command:`conan hello moon`.
     Notice that to declare a sub-command is required an empty Python function acts as the main command.
 
 
+Formatters
+++++++++++
+
+The formatters are functions that receive the returned result from the command as argument, as seen above.
+A command can have as many formatters as desired, but if these formatters need different parameters, then it can become
```

Comment:
Would nice to mention the "text" formatter is required:)

Reply:
Done